### ISSUE TITLE
Question: Caching

### ISSUE THREAD

**Issue by ganySA** [Breeze/breeze.bridge2.angular#10]
Hi

Just a question - are you aware of any query cache libraries that work with breeze JS. 

Something like: ngx-cacheable?

I would like to control how underlying http queries are cached based on a time - so when i query using entity manager if the underlying http request is within the cache period it can be simply use the cached values with an override to bypass cache and force request.

I have queries based on date + other ID paramters, and i am trying not to re-invent the wheel and build my on cache based on parameters.

Thanks

**Comment by marcelgood** [comment]
No, not aware of anything, but this is not very hard to do using an `HttpInterceptor`. You can intercept your Breeze queries, cache the response and return cached responses.
